CONTRIBUTING.md

```diff
@@ -3,296 +3,73 @@ Contributing to Flutter
-document for more details), submit your changes to the `master` branch using one of
-the following methods:
-
-* Wait for one of the project maintainers to submit it for you.
-* Click the green "Merge pull request" button on the GitHub UI of your pull
-  request (requires commit access)
-
-You must complete the
-[Contributor License Agreement](https://cla.developers.google.com/clas).
-You can do this online, and it only takes a minute.
-If you've never submitted code before, you must add your (or your
-organization's) name and contact info to the [AUTHORS](AUTHORS) file.
-
-We grant commit access to people who have gained our trust and demonstrated
-a commitment to Flutter.
-
-Tools for tracking and improving test coverage
-----------------------------------------------
-
-We strive for a high degree of test coverage for the Flutter framework. We use
-Coveralls to [track our test coverage](https://coveralls.io/github/flutter/flutter?branch=master).
-You can download our current coverage data from cloud storage and visualize it
-in Atom as follows:
-
- * Install [Atom](https://atom.io/).
- * Install the [lcov-info](https://atom.io/packages/lcov-info) package for Atom.
- * Open the `packages/flutter` folder in Atom.
- * Open a Dart file in the `lib` directory an type `Ctrl+Alt+C` to bring up the
-   coverage data.
-
-If you don't see any coverage data, check that you have an `lcov.info` file in
-the `packages/flutter/coverage` directory. It should have been downloaded by the
-`flutter update-packages` command you ran previously.
-
-If you want to iterate quickly on improving test coverage, consider using this
-workflow:
-
- * Open a file and observe that some line is untested.
- * Write a test that exercises that line.
- * Run `flutter test --merge-coverage path/to/your/test_test.dart`.
- * After the test passes, observe that the line is now tested.
-
-This workflow merges the coverage data from this test run with the base coverage
-data downloaded by `flutter update-packages`.
-
-See [issue 4719](https://github.com/flutter/flutter/issues/4719) for ideas about
-how to improve this workflow.
-
-Communication
--------------
+We invite you to join our team! Everyone is welcome to contribute code
+via pull requests, to file issues on GitHub, to help people asking for
+help on our mailing lists or on Stack Overflow, to help triage,
+reproduce, or fix bugs that people have filed, to add to our
+documentation, or to help out in any other way.
+
+We grant commit access (which includes full rights to the issue
+database, such as being able to edit labels) to people who have gained
+our trust and demonstrated a commitment to Flutter.
+
+This document focuses on what is needed to contribute by writing code
+and submitting pull requests for the Flutter framework. For
+information on contributing in other ways, see [the CONTRIBUTING page
+on flutter.io](https://flutter.io/CONTRIBUTING).
```

Comment:
This is a 404 right now.

Reply:
fixed. This got renamed to https://flutter.io/community recently. Thanks.